tests/unittest/_torch/modules/moe/test_moe_module.py

```
@@ -723,6 +723,97 @@ def run_forward():
+    # survives between cases. NVLINK one-sided comm keeps a symmetric-memory
+    # workspace singleton that assumes a single workspace size per process
+    # lifetime, so a later case with a different shape would otherwise reuse a
+    # mismatched workspace. Reset it after every case to match the fresh-process
+    # semantics of the original per-case executor.
+    import gc as _gc
+
+    try:
+        from tensorrt_llm._torch.modules.fused_moe.communication.nvlink_one_sided import (
+            NVLinkOneSided as _NVOS,
+        )
+    except Exception:
```

Ruff (0.15.20)

[warning] Do not catch blind exception: `Exception`

(BLE001)